Libplanet.Extensions.Cocona/Commands/StoreQueryCommand.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.Collections.Immutable;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Bencodex.Types;
+using Cocona;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Tx;
+
+namespace Libplanet.Extensions.Cocona.Commands
+{
+    public class StoreQueryCommand
+    {
+        [Command(Description = "Query block by the index")]
+        public void BlockByIndex(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("BLOCK-INDEX", Description = "block index")]
+            int blockIndex
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            var chainId = store.GetCanonicalChainId();
+            if (chainId == null)
+            {
+                Console.Error.WriteLine("cannot find the main branch of the blockchain");
+                return;
+            }
+
+            var blockHash = store.IndexBlockHash(chainId.Value, blockIndex);
+            if (blockHash == null)
+            {
+                Console.Error.WriteLine($"cannot find the block with the height[{blockIndex}]" +
+                                        $" within the blockchain[{chainId}]");
+                return;
+            }
+
+            var block = store.GetBlock<BareAction>(blockHash.Value);
+            if (block == null)
+            {
+                Console.Error.WriteLine($"cannot find the block with the hash[{blockHash}]");
+            }
+
+            Console.WriteLine(SerializeHumanReadable(block));
+        }
+
+        [Command(Description = "Query block by the hash")]
+        public void BlockByHash(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("BLOCK-HASH", Description = "block hash")]
+            string blockHash
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            var block = store.GetBlock<BareAction>(BlockHash.FromString(blockHash));
```

Comment:
Not very important but we often use when need dummy action like this, we named it `DummyAction`

Reply:
Renamed. https://github.com/planetarium/libplanet/pull/1298/commits/7270d416317506b5311866a80eeb2044470eec93